README.md

```diff
@@ -68,6 +68,15 @@ Please note that the date shown is the date that the change was released to the
 
 ---
 
+### 15 April 2024
+
+The following changes are now available in the Sandbox:
+
+#### self-employment-business-api
+
+- Added 2 new optional fields transitionProfitAmount, transitionProfitAccelerationAmount in annualAdjustmentsType object for Retrieve a Self-Employment Annual Submission and Create and Amend Self-Employment Annual Submission Endpoints.
+- Added new error code 400 (RULE_WRONG_TPA_AMOUNT_SUBMITTED) for Create and Amend Self-Employment Annual Submission Endpoint.
```

Comment:
Similarly, plus the error code is usually the 'RULE_WRONG_TPA_AMOUNT_SUBMITTED' bit:
```suggestion
- Added new error code RULE_WRONG_TPA_AMOUNT_SUBMITTED for `v3.0` `Create and Amend Self-Employment Annual Submission Endpoint`.
```